boards/mediatek/mt8188/mt8188_adsp.dts

```diff
@@ -83,7 +83,7 @@
 		};
 
 		/* Generated code for AFE devices */
-		#include "afe-mt8188.dts"
+		#include "afe-mt8188.dtsi"
```

Comment:
shouldn't be indented

Reply:
Includes in nodes have always been indented 

```
/ {
    node {
        #include .....
    };
};
```

See https://github.com/zephyrproject-rtos/zephyr/blob/0601335d85ed22439671a25a23b8d550fb4893fd/dts/arm/nordic/nrf5340_cpuapp.dtsi#L116 

the above is one of many cases like this 

If we want to change this rule I can but it will be in a separate PR